ocs_ci/utility/ibmcloud.py

```diff
@@ -265,15 +265,12 @@ def restart_nodes(self, nodes, timeout=900, wait=True):
         provider_id = nodes[0].get()["spec"]["providerID"]
         cluster_id = provider_id.split("/")[5]
 
-        cmd = f"ibmcloud ks workers --cluster {cluster_id} --output json"
-        out = run_ibmcloud_cmd(cmd)
-        worker_nodes = json.loads(out)
-
-        if len(worker_nodes) > 0:
-            for node in worker_nodes:
-                cmd = f"ibmcloud ks worker reboot --cluster {cluster_id} --worker {node['id']} -f"
-                out = run_ibmcloud_cmd(cmd)
-                logger.info(f"Node restart command output: {out}")
+        for node in nodes:
+            provider_id = node.get()["spec"]["providerID"]
```

Comment:
Is this same as the provider_id which is already assigned in line 265 ?

Reply:
@gayathrimenath-ibm Could you please check this comment.

Reply:
@gayathrimenath-ibm can you fix this please?